jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -459,6 +463,10 @@ def loadServerlessConfig(String runtime, def isEventSchdld, def isScheduleEnable
 		addEvents(isScheduleEnabled, isEc2EventEnabled, isS3EventEnabled, isStreamEnabled, isDynamoDbEnabled)
 	}
 
+	if (service == 'cloud-logs-streamer') {
```

Comment:
change service to config['service']  and also check its domain as "jazz"

Reply:
Also are you sure there is no change in the serverless.yml files ?

Reply:
Done.